demo/kubernetes/rook-operator.yaml

```diff
@@ -12,5 +12,9 @@ spec:
     spec:
       containers:
       - name: rook-operator
-        image: quay.io/rook/rook-operator:dev-2017-02-17-k8s
-        args: ["--container-version=dev-2017-02-17-k8s"]
+        image: quay.io/rook/rook-operator:dev-2017-02-23-k8s
+        env:
+        - name: ROOK_OPERATOR_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
```

Comment:
im also not too clear about the namespace strategy. Could you quickly explain in which namespace you envision all the resources to be created? Including the rook-operator.